### PR TITLE
Post/compatibility table

### PR DESCRIPTION
**Pull request by etagwerker**
Hi @ercohen14, 

I had to revert #24 because it was missing information. I'll add comments below so that you can address them on Monday.

Thanks!

### CODE REVIEW

File: _posts/2019-04-26-compatibility-table.md [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+      <td>2.0</td>
+      <td>&gt;= 1.3.6</td>
+      <td>EOL</td>
+    </tr>
+    <tr class="eol">
+      <td>0.8.0 to 3.2.13.rc2</td>
+      <td>1.8.7</td>
+      <td></td>
+      <td>&gt;= 1.3.6</td>
+      <td>EOL</td>
```
Comment (etagwerker) [inline _posts/2019-04-26-compatibility-table.md]: @ercohen14 Source? We want to know the original source that says that Rails 3.2 was "end of life'ed"

File: _posts/2019-04-26-compatibility-table.md [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+      <td></td>
+      <td>&gt;= 1.8.11</td>
+      <td>EOL</td>
+    </tr>
+    <tr class="eol">
+      <td>4.0.0 to 4.0.x</td>
+      <td>&gt;= 1.8.7</td>
+      <td></td>
+      <td>&gt;= 1.8.11</td>
+      <td>EOL</td>
```
Comment (etagwerker) [inline _posts/2019-04-26-compatibility-table.md]: @ercohen14 Source?

File: _posts/2019-04-26-compatibility-table.md [hunk 3]
```diff
@@ -0,0 +1,97 @@
...
+      <td>2.1</td>
+      <td>&gt;= 1.8.11</td>
+      <td>EOL</td>
+    </tr>
+    <tr class="eol">
+      <td>4.0.5 to 4.1.0.rc2</td>
+      <td>&gt;= 1.9.3</td>
+      <td></td>
+      <td>&gt;= 1.8.11</td>
+      <td>EOL</td>
```
Comment (etagwerker) [inline _posts/2019-04-26-compatibility-table.md]: @ercohen14 Source?

File: _posts/2019-04-26-compatibility-table.md (hunk 1 above)
Comment (ercohen14) [reply _posts/2019-04-26-compatibility-table.md]: The column title is a link to an article that lists all of the versions of Rails that are being maintained or not.

File: _posts/2019-04-26-compatibility-table.md (hunk 3 above)
Comment (etagwerker) [reply _posts/2019-04-26-compatibility-table.md]: I believe the source for this one would be this page: https://weblog.rubyonrails.org/2016/6/30/Rails-5-0-final/

> Note: As per our maintenance policy, the release of Rails 5.0 will mean that bug fixes will only apply to 5.0.x, regular security issues to 5.0.x and 4.2.x, and severe security issues also to 5.0.x and 4.2.x (but when 5.1 drops, to 5.1.x, 5.0.x, and 4.2.x). This means 4.1.x and below will essentially be unsupported! Ruby 2.2.2+ is now also the only supported version of Rails 5.0+.

**Pull request merged**